adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -240,7 +240,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord])
   def saveAsSamString(sd: SequenceDictionary,
                       rgd: RecordGroupDictionary): String = {
     // convert the records
-    val (convertRecords: RDD[SAMRecordWritable], header: SAMFileHeader) = rdd.convertToSam(sd, rgd)
+    val (convertRecords: RDD[SAMRecordWritable], header: SAMFileHeader) = convertToSam(sd, rgd)
```

Comment:
right. I'd not considered this before, but having now thought about it, I think it actually makes sense to proactively not use that implicit to create another `AlignmentRecordRDDFunctions` wrapping the same RDD when we're already inside a `AlignmentRecordRDDFunctions` that wraps that RDD.

this stylistic point has broader appeal to me, but it's a minor one overall and as discussed elsewhere there are a few ways to get approximately the same outcome here.